examples/pom.xml

```diff
@@ -63,7 +63,7 @@
         
         <seata.version>1.4.2</seata.version>
         
-        <junit.version>4.12</junit.version>
+        <junit.version>4.13.2</junit.version>
```

Comment:
`assertThat` is marked as deprecated in junit 4.13. This require further consideration.

Reply:
Maybe https://shardingsphere.apache.org/community/en/contribute/code-conduct/ should be updated?

Reply:
I reverted the junit piece - there is a CVE but it could dealt with in a separate issue

Reply:
Refer to https://github.com/junit-team/junit4/pull/1150 , if the version is raised, it means a PR with keyword changes. (Similar to `org.junit.Assert.assertThat` is replaced by `org.hamcrest.MatcherAssert.assertThat` )